code-challenges.test.js

```diff
@@ -13,61 +13,134 @@
-
-const studyMinutesWeek1 = {
-  sunday: 90,
-  monday: 30,
-  tuesday: 20,
-  wednesday: 15,
-  thursday: 30,
-  friday: 15,
-  saturday: 60
-}
-// Expected output: [15, 15, 20, 30, 30, 60, 90]
-
-const studyMinutesWeek2 = {
-  sunday: 100,
-  monday: 10,
-  tuesday: 45,
-  wednesday: 60,
-  thursday: 20,
-  friday: 15,
-  saturday: 65
-}
-// Expected output: [10, 15, 20, 45, 60, 65, 100]
-
-// b) Create the function that makes the test pass.
-
-// Pseudo code:
-
-// --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+//input: object
+//output: [15, 15, 20, 30, 30, 60, 90], [10, 15, 20, 45, 60, 65, 100]
+// describe("reSort", () => {
+//   it("takes in an object and returns an array of the values sorted from least to greatest", () => {
+//     const studyMinutesWeek1 = {
+//       sunday: 90,
+//       monday: 30,
+//       tuesday: 20,
+//       wednesday: 15,
+//       thursday: 30,
+//       friday: 15,
+//       saturday: 60
+//     }
+//     // Expected output: [15, 15, 20, 30, 30, 60, 90]
+    
+//     const studyMinutesWeek2 = {
+//       sunday: 100,
+//       monday: 10,
+//       tuesday: 45,
+//       wednesday: 60,
+//       thursday: 20,
+//       friday: 15,
+//       saturday: 65
+//     }
+//     expect(reSort(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90])
+//     expect(reSort(studyMinutesWeek2)).toEqual([10, 15, 20, 45, 60, 65, 100])
+//   })
+// })
+//     // Expected output: [10, 15, 20, 45, 60, 65, 100]
+// // b) Create the function that makes the test pass.
+
+// // Pseudo code:
+// // create a new variable 
+// // that takes in object as input to extract the values from the object, sorts them using sort method .sort()
+// // use (a, b) => a - b for sorting
+// //
+// // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+
+// const reSort = (obj) => {
+//   const valuesArray = Object.values(obj)
+//   valuesArray.sort((a, b) => a - b)
+//   return valuesArray
+// }
 
 // a) Create a test with expect statements for each of the variables provided.
+//input: [100, -17, -23, -9], [250, -89, 100, -96], []
+//output: [100, 83, 60, 51], [250, 161, 261, 165], []
 
-const accountTransactions1 = [100, -17, -23, -9]
-// Expected output: [100, 83, 60, 51]
 
-const accountTransactions2 = [250, -89, 100, -96]
-// Expected output: [250, 161, 261, 165]
+describe("addMe", () => {
+it("add each number from each other", () => {
+  const accountTransactions1 = [100, -17, -23, -9]
+  // Expected output: [100, 83, 60, 51]
+  const accountTransactions2 = [250, -89, 100, -96]
+  // Expected output: [250, 161, 261, 165]
+  const accountTransactions3 = []
+  // Expected output: []
+  expect(addMe(accountTransactions1)).toEqual[100, -17, -23, -9]
+  expect(addMe(accountTransactions2)).toEqual[250, -89, 100, -96]
```

Comment:
Remember that `.toEqual` requires parentheses. The way it is set up here, it's not testing for anything so anything will pass. Also, these are not the expected outputs you would want to put here, make sure to keep attention to detail.